### ISSUE TITLE
Build public images?

### ISSUE THREAD

**Issue by ncoghlan** [bkabrda/anitya-docker#3]
Reading https://seven.centos.org/2016/11/introducing-centos-container-image-scanners/ made me wonder if we should experiment with building public images here, and then submit a PR to enable that to the upstream Anitya repo as well.

**Comment by bkabrda** [comment]
:+1: nice, I'll definitely try this out.

**Comment by bkabrda** [comment]
Hmm, on second thought, I'd like to wait until we have the patches for API authentication. Adding image with an insecure application to the container index doesn't feel right... WDYT?

**Comment by ncoghlan** [comment]
Yeah, fair point, although once we have that, we won't be far off getting the changes merged back upstream and hence potentially eligible for building in Fedora instead :)

**Comment by mohammedzee1000** [comment]
Hey, we definitely would love to have you on the container index as well.
